docs/starters.yml

```diff
@@ -28,6 +28,15 @@
     - Offline support
     - Web App Manifest
     - SEO
+- url: https://gatsby-tailwind-emotion-starter.netlify.com/
+  repo: https://github.com/muhajirframe/gatsby-tailwind-emotion-starter
+  description: n/a
```

Comment:
I know many entries has `n/a` description, but could you add some 1-line description?

Reply:
sure